vector/src/main/java/im/vector/app/core/epoxy/charsequence/EpoxyCharSequence.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.core.epoxy.charsequence
+
+/**
+ * Wrapper for a CharSequence, which support mutation of the CharSequence, which can happen during rendering
+ */
+class EpoxyCharSequence(val charSequence: CharSequence) {
```

Comment:
Maybe let the EpoxyCharSequence inherits CharSequence like this, to avoid accessing charSequence value everywhere 

```class EpoxyCharSequence(val charSequence: CharSequence) : CharSequence by charSequence```

Reply:
Good idea, let me try it.

Reply:
It does not work, the TextView displays the Object and not the CharSequence... Weird

<img width="312" alt="image" src="https://user-images.githubusercontent.com/3940906/147946407-52805221-2ee5-44be-ba0c-ee25aca1b98a.png">

Reply:
According to @ganfra investigation, it can work using
```kotlin
class EpoxyCharSequence(val charSequence: CharSequence) : Spannable by SpannableString(charSequence) {...}
```
but I prefer not to create a new SpannableString for each charSequence.
Using `.charSequence` whenever it is necesssary is probably better, because it forces the developer to do it (no "hidden magic")